maa-cli/src/main.rs

```diff
@@ -226,6 +226,12 @@ enum SubCommand {
         #[arg(long)]
         timezone: Option<i8>,
     },
+    /// Clearing the caches of maa-cli and maa core
+    Cleanup {
+        /// Does not ask for deletion
+        #[clap(short, long, value_delimiter = ' ', num_args = 1..)]
+        specify: Option<Vec<String>>,
```

Comment:
The target to clear can be command arguments instead of option arguments.
```suggestion
        targets: Vec<CleanupTarget>,
```

IMO, A `ValueEnum` `CleanupTarget` instead of raw `String` will be better.

BTW, if you think inquiry before is useful, an `inquiry` option might also be include.

Reply:
It is necessary to make a interruption to let the user know which items will be deleted, so `inquiry` is not required.